src/components/config_profile/test/profile_test.cc

```diff
@@ -724,6 +724,193 @@ TEST_F(ProfileTest, CheckIntContainerInSecurityData) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial max_supported_protocol_version
+  EXPECT_TRUE(profile_.enable_protocol_4());
+}
+
+TEST_F(ProfileTest, CheckAppIconsFolderMaxSize) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial profile_.app_icons_folder_max_size
+  EXPECT_EQ(104857600u, profile_.app_icons_folder_max_size());
+}
+
+TEST_F(ProfileTest, CheckAppIconsAmountToRemove) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial app_icons_amount_to_remove
+  EXPECT_EQ(1u, profile_.app_icons_amount_to_remove());
+}
+
+TEST_F(ProfileTest, CheckMaximumControlPayloadSize) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial maximum_control_payload_size
+  EXPECT_EQ(0u, profile_.maximum_control_payload_size());
+}
+
+TEST_F(ProfileTest, CheckMaximumRpcPayloadSize) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial maximum_rpc_payload_size
+  EXPECT_EQ(0u, profile_.maximum_rpc_payload_size());
+}
+
+TEST_F(ProfileTest, CheckMaximumAudioPayloadSize) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial maximum_audio_payload_size
+  EXPECT_EQ(0u, profile_.maximum_audio_payload_size());
+}
+
+TEST_F(ProfileTest, CheckMaximumVideoPayLoadSize) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial maximum_video_payload_size
+  EXPECT_EQ(0u, profile_.maximum_video_payload_size());
+}
+
+TEST_F(ProfileTest, CheckHmiCapabilitiesFileName) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial hmi_capabilities_file_name
+  EXPECT_EQ(profile_.app_config_folder() + "/" + "hmi_capabilities.json",
+            profile_.hmi_capabilities_file_name());
+}
+
+TEST_F(ProfileTest, CheckDefaultTimeout) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
+  EXPECT_EQ("smartDeviceLink_test.ini", profile_.config_file_name());
+  // Check initial default_timeout
+  EXPECT_EQ(10000u, profile_.default_timeout());
+}
+
+TEST_F(ProfileTest, CheckAppResumingTimeout) {
+  EXPECT_EQ(3000u, profile_.app_resuming_timeout());
```

Comment:
@Ypostolov Where these values came from?

Reply:
@AByzhynar 
fixed in https://github.com/smartdevicelink/sdl_core/pull/1995/commits/8b04ba878df1f894f8688d2196f7a6c4b29b9328